### PR TITLE
Add AnswerPossibilityGroup

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -1,124 +1,149 @@
...
-    attributes:
-      administrator:
-        email: E-Mail
-        password: Passwort
-        password_confirmation: Passwort-Wiederholung
+      question_group:
+        one: Fragengruppe
+        other: Fragengruppen
       survey:
-        title_de: Deutscher Titel
```
Comment (lukasbischof) [inline config/locales/de.yml]: Why did you delete all those activerecord translations?

File: db/migrate/20200110133403_create_answer_possibility_groups.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateAnswerPossibilityGroups < ActiveRecord::Migration[6.0]
+  def change
+    create_table :answer_possibility_groups do |t|
+      t.string :description
```
Comment (lukasbischof) [inline db/migrate/20200110133403_create_answer_possibility_groups.rb]: Guess a description is required, so require it also on db level

File: spec/factories/answer_possibility_groups.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :answer_possibility_group do
+  end
```
Comment (lukasbischof) [inline spec/factories/answer_possibility_groups.rb]: Missing 
```suggestion
    description { 'Some description' }
  end
```

File: spec/models/answer_possibility_group_spec.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AnswerPossibilityGroup, type: :model do
+  it { is_expected.to have_many :answer_possibilities }
```
Comment (lukasbischof) [inline spec/models/answer_possibility_group_spec.rb]: Plz add also spec for required description attribute

File: app/views/fields/has_many/_show.html.erb [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<%#
```
Comment (lukasbischof) [inline app/views/fields/has_many/_show.html.erb]: What exactly is different in this file to the original? I cannot find something...

File: config/locales/de.yml (hunk 1 above)
Comment (TheFehr) [reply config/locales/de.yml]: Not deleted moved. Far down.
